luaui/barwidgets.lua

```diff
@@ -1495,23 +1495,24 @@ function widgetHandler:WidgetAt(x, y)
 end
 
 function widgetHandler:MousePress(x, y, button)
-	local mo = self.mouseOwner
```

Comment:
Same behavior, just simplified. Only difference is we allow `MousePress2` to be triggered.